kernel/os/src/arch/cortex_m4/os_fault.c

```diff
@@ -76,7 +76,11 @@ struct coredump_regs {
     uint32_t psr;
 };
 
-#if MYNEWT_VAL(OS_COREDUMP)
+#if MYNEWT_VAL(OS_COREDUMP_CB)
+static coredump_cb_t cd_cb = NULL;
```

Comment:
Don’t need to explicitly set it to NULL since it already is in the BSS

Reply:
I would name it as g_coredump_cb

Reply:
Renamed